src/components/layout/yearInReview/2021/byTheNumbers.tsx

```diff
@@ -44,56 +54,98 @@ const ByTheNumbers: React.FC = () => {
               />
             }
           >
-            <b>Total unique page views</b> for all projects
+            <FormattedMessage
+              id="page.year-in-review.2021.section.by-the-numbers.statistic.0"
+              defaultMessage="<b>Total unique page views</b> for all projects"
+              values={{
+                b: (chunks) => <b>{chunks}</b>,
+              }}
+            />
           </Number>
           <div className="flex flex-col md:flex-row flex-wrap mt-5">
             <Number
               number={
                 <AnimatedNumber number={13100} className="text-green" approx />
               }
             >
-              Tweets by our <b>5 Minutes 5 Vegans</b> support bot
+              <FormattedMessage
+                id="page.year-in-review.2021.section.by-the-numbers.statistic.1"
+                defaultMessage="Tweets by our <b>5 Minutes 5 Vegans</b> support bot"
+                values={{
+                  b: (chunks) => <b>{chunks}</b>,
+                }}
+              />
             </Number>
             <Number
               number={
                 <AnimatedNumber number={93432} className="text-green" approx />
               }
             >
-              Clicks directing activists to orgs{' '}
-              <Link href="https://veganactivism.org">
-                <a>
-                  <b>veganactivism.org</b>
-                </a>
-              </Link>
+              <FormattedMessage
+                id="page.year-in-review.2021.section.by-the-numbers.statistic.2"
+                defaultMessage="Clicks directing activists to orgs {link}"
+                values={{
+                  link: (
+                    <Link href="https://veganactivism.org">
+                      <a>
+                        <b>veganactivism.org</b>
+                      </a>
+                    </Link>
+                  ),
+                }}
+              />
             </Number>
             <Number
               number={
                 <AnimatedNumber number={500000} className="text-green" approx />
               }
             >
-              Unique page visits for <b>Vegan Bootcamp</b>
+              <FormattedMessage
+                id="page.year-in-review.2021.section.by-the-numbers.statistic.3"
+                defaultMessage="Unique page visits for {project}"
+                values={{
+                  project: <b>Vegan Bootcamp</b>,
```

Comment:
Same as above. Text should always be inside the defaultMessage, to keep it configurable by translators.

Reply:
I am a bit confused by this. As `Vegan Bootcamp` is an organisation, don't we want the translators to ignore it anyway? IIRC I've hopefully only included text inside the `values` if we don't want it translated anyway.

Reply:
Let's say Vegan Bootcamp is renamed (hypothetically, highly unlikely but not impossible), we'd have to update this inside the code. Instead I'd prefer all text managed in one source which would be the json files/weblate.